doc/release-notes.md

```diff
@@ -26,16 +26,27 @@ likely require a reindex.
 
 # Release Notes
 
-v2 P2P Downgrade Issues
+Deployment and CI Fixes
 -----------------------
 
-This version addressed a problem affecting certain Dash-specific connection types, including mixing and masternode probes, when
-downgrading from v2 to v1 connections. This resulted in increased connection count and load for masternodes. (dash#6574)
+- Reverted deployment containers and CI workflows to Ubuntu 22.04 LTS (`jammy`) due to regressions in Ubuntu 24.04 (`noble`) causing build failures.
+- Pinned QEMU version to avoid segmentation faults during container builds (dash#6586).
```

Comment:
#6586 is no longer a part of this PR it seems, might want to drop the whole `Deployment and CI Fixes` section if that's intentional

Reply:
Oops unintentional!